db/migrate/20221129094342_create_value_definitions.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+# Data model for storing value definitions
+class CreateValueDefinitions < ActiveRecord::Migration[7.0]
+  def change
+    create_table :value_definitions, id: :uuid do |t|
+      t.string :ref_id
+      t.string :title
+      t.text :description
+      t.string :value_type
+      t.string :default_value
+      t.numeric :lower_bound, default: nil
+      t.numeric :upper_bound, default: nil
+      t.uuid :benchmark_id
+      t.references :benchmark_id, type: :uuid, index: true
```

Comment:
1. Why you have them twice?
2. Why are you referencing to `benchmark_id`?

https://github.com/RedHatInsights/compliance-backend/blob/master/db/migrate/20220202002459_create_rule_groups.rb#L10